analysis_driver/driver.py

```diff
@@ -254,6 +254,10 @@ def bcbio_var_calling_pipeline(dataset):
     gender_validation = qc.GenderValidation(dataset, sample_dir, vcf_file)
     gender_validation.start()
 
+    #vcf stats
+    vcf_stats = qc.VCFStats(dataset, sample_dir, vcf_file)
```

Comment:
`VCFStats` is not imported in `quality_control.__init__.py`, so this will break.